config/settings_schema.json

```diff
@@ -7,6 +7,201 @@
+      },
+      {
+        "type": "header",
+        "content": "t:settings_schema.global.settings.header.border.content"
+      },
+      {
+        "type": "range",
+        "id": "card_border_width",
+        "min": 0,
+        "max": 10,
+        "step": 1,
+        "unit": "px",
+        "label": "t:settings_schema.global.settings.width.label",
+        "default": 1
+      },
+      {
+        "type": "range",
+        "id": "card_border_opacity",
+        "min": 0,
+        "max": 100,
+        "step": 5,
+        "unit": "%",
+        "label": "t:settings_schema.global.settings.opacity.label",
+        "default": 10
+      },
+      {
+        "type": "header",
+        "content": "t:settings_schema.global.settings.header.shadow.content"
+      },
+      {
+        "type": "range",
+        "id": "card_shadow_opacity",
+        "min": 0,
+        "max": 100,
+        "step": 5,
+        "unit": "%",
+        "label": "t:settings_schema.global.settings.opacity.label",
+        "default": 10
+      },
+      {
+        "type": "range",
+        "id": "card_shadow_horizontal_offset",
+        "min": -20,
+        "max": 20,
+        "step": 2,
+        "unit": "px",
+        "label": "t:settings_schema.global.settings.horizontal_offset.label",
+        "default": 0
+      },
+      {
+        "type": "range",
+        "id": "card_shadow_vertical_offset",
+        "min": -20,
+        "max": 20,
+        "step": 2,
+        "unit": "px",
+        "label": "t:settings_schema.global.settings.vertical_offset.label",
+        "default": 0
+      },
+      {
+        "type": "range",
+        "id": "card_shadow_blur_radius",
+        "min": 0,
+        "max": 20,
+        "step": 2,
+        "unit": "px",
+        "label": "t:settings_schema.global.settings.blur_radius.label",
+        "default": 0
+      },
+      {
+        "type": "select",
+        "id": "card_badge_position",
+        "options": [
+          {
+            "value": "bottom left",
+            "label": "t:settings_schema.cards.settings.card_badge_position.options__1.label"
+          },
+          {
+            "value": "bottom right",
+            "label": "t:settings_schema.cards.settings.card_badge_position.options__2.label"
+          },
+          {
+            "value": "top left",
+            "label": "t:settings_schema.cards.settings.card_badge_position.options__3.label"
+          },
+          {
+            "value": "top right",
+            "label": "t:settings_schema.cards.settings.card_badge_position.options__4.label"
+          }
+        ],
+        "default": "bottom left",
+        "label": "t:settings_schema.cards.settings.card_badge_position.label"
+      },
+      {
+        "type": "range",
+        "id": "card_badge_corner_radius",
+        "min": 0,
+        "max": 10,
+        "step": 1,
+        "unit": "px",
+        "label": "t:settings_schema.global.settings.corner_radius.label",
+        "default": 10
+      },
```

Comment:
Below the Badge `Corner radius` setting we should bring the [Badge color scheme](https://screenshot.click/21-11-s5cqn-ciery.png) from the `Style` setting to optimize the IA of similar settings/topic.